### PR TITLE
Fixed docker image env variable dependencies

### PR DESCRIPTION
**Pull request by Kingdawnage**
A text file with the id name and the content being the value of the variable is required In order for it to be passed to the docker build. More context here
https://docs.docker.com/build/building/secrets/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,10 +24,15 @@ COPY . .
 ENV NODE_ENV=production
 ENV NEXT_TELEMETRY_DISABLED 1
 # todo: add here after adding supabase
-# ENV NEXT_PUBLIC_SUPABASE_URL
```
Comment (devhik0) [inline Dockerfile:27]: Revert this file and just write secrets here as ENV.
Also change system permissions and add user foodie and give related permissions, don't add new line just edit current ones.

**Review by devhik0 — CHANGES_REQUESTED** [review]
Revert this file and just write secrets here as ENV.
Also change system permissions and add user foodie and give related permissions, don't add new line just edit current ones.

File: Dockerfile (hunk 1 above)
Comment (Kingdawnage) [reply Dockerfile:27]: But the secrets will be exposed in the docker image

Comment (devhik0) [reply Dockerfile:27]: They're just public keys, just put them as is

**Review by devhik0 — APPROVED** [review]

**Pull request merged**